### DISCUSSION TITLE
Batch Script插件执行结果中文乱码

### DISCUSSION THREAD

**Discussion by FangPengbo** [Tencent/bk-ci#7226] [Q&A]
agent节点机是windows英文版的系统，默认编码是windows-1252，
使用batch插件执行命令，输出结果包含中文的话，展示出来的结果会出现乱码
这个有解决的办法吗？除了修改节点机的系统。

**Comment by hummerstudio** [comment]
最近刚刚遇到过同样的问题，可以这样配置，**不会改变系统英文语言**，可以试试：
Windows设置-时间和语言-右侧“日期、时间和区域格式设置”-右侧“其他日期、时间和区域设置”-"更改日期、时间或数字格式"-切换到"管理"标签-非 Unicode 程序的语言”更改区域设置“，修改为中文(简体，中国)
![image](https://user-images.githubusercontent.com/7810874/178405026-b70d2e1b-c80c-4f74-8d19-a55b96dcd939.png)

**Reply by FangPengbo** [reply]
好的，我们尝试一下，谢谢。
